src/containers/DebugContainer.tsx

```diff
@@ -0,0 +1,13 @@
+import { createContainer } from 'unstated-next'
+
+const useDebugContainer = () => {
+  const DEBUG_LOGIN = false
+  const DEBUG_AUTH = false
```

Comment:
These are turned off now, but may be useful when trying to improve the token code further.